frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -77,9 +82,84 @@ pods:
         goal: FINISHED
         cmd: ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/nodetool cleanup -p {{TASKCFG_ALL_JMX_PORT}} -- $CASSANDRA_KEYSPACE_TABLES
         resource-set: sidecar-resources
+      backup-schema:
+        goal: FINISHED
+        cmd: ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/cqlsh --cqlversion="3.4.0" -e "DESC SCHEMA" node-0-server.{{SERVICE_NAME}}.mesos > container-path/schema.cql
```

Comment:
Also, this one relies on `node-0-server`, I'm just wondering if this will ever leave somebody in a state where they cannot backup data, because `node-0-server` cannot be brought up ?